semester1/DT/oefenzitting/oefz1.pl

```diff
@@ -0,0 +1,126 @@
+father(anton,bart).
+father(anton,daan).
+father(anton,elisa).
+father(fabian,anton).
+
+mother(celine,bart).
+mother(celine,daan).
+mother(celine,gerda).
+mother(gerda,hendrik).
+
+/* X \== Y: kan X een andere waarde krijgen dan Y? */
```

Comment:
Het is geen echte vraag, de commentaar is bedoeld om aan te geven dat X \== Y gaat controleren of X een andere waarde kan krijgen dan Y. Deze comment is dus louter informatief.